UpSkill_App/UpSkill.Infrastructure/Data/Models/Coach.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace UpSkill.Infrastructure.Data.Models
+{
+    using System.Linq;
+
+    public class Coach : ApplicationUser
```

Comment:
I thought we decided not to extend ApplicationUser, but to have it as a Navigation Property within our class, with the AppUserId as a Foreign Key. Did I get something wrong, or maybe you, guys, decided something different, since we last talked?

Reply:
I read some articles about when inheritance should be used and I decided that this is one of those cases. If you have another opinion I would be more than happy to discuss it in the daily meeting tomorrow 😃

Reply:
Can I have the link to what you read on the topic, please?